code/datums/components/spawner.dm

```diff
@@ -15,8 +15,6 @@
 	COOLDOWN_DECLARE(spawn_delay)
 
 /datum/component/spawner/Initialize(spawn_types = list(), spawn_time = 30 SECONDS, max_spawned = 5, faction = list(FACTION_MINING), spawn_text = null)
-	if (!islist(spawn_types))
-		CRASH("invalid spawn_types to spawn specified for spawner component!")
```

Comment:
this is just checking for it being a list, not it having any items in it

Reply:
yeah now that i think about it, this PR shouldn't have been a thing since it was meant to be fixed six months ago in #73976. why didn't that work?

Reply:
It didn't work because the first time the component processes it calls pick on the lost which will runtime if its empty
I guess I didn't need to fe,lve this check though